lib/jsonld-signatures.js

```diff
@@ -263,12 +336,12 @@ api.verify = util.callbackify(async function(input, options) {
       //const Suite = require('./suites/' + algorithm);
       const Suite = suites[algorithm];
       const verified = await new Suite(injector).verify(
-        f, Object.assign({}, options, {framed}));
+        f, Object.assign({}, options, {expanded, framed, purpose}));
```

Comment:
Do we still need `expanded` at this point?  Relates to the comments above on L268.